x-pack/plugins/apm/common/utils/to_kuery_filter_format.ts

```diff
@@ -5,10 +5,34 @@
  * 2.0.
  */
 
+import { isEmpty } from 'lodash';
+
+type Separator = 'OR' | 'AND';
+
 export function toKueryFilterFormat(
   key: string,
   values: string[],
-  separator: 'OR' | 'AND' = 'OR'
+  separator: Separator = 'OR'
 ) {
   return values.map((value) => `${key} : "${value}"`).join(` ${separator} `);
 }
+export function mergeKueries(
```

Comment:
Might be better to move this function into its own file or rename the current file to something more generic